package.json

```diff
@@ -67,7 +67,8 @@
   "lavamoat": {
     "allowScripts": {
       "@lavamoat/preinstall-always-fail": false,
-      "keccak": true
+      "keccak": true,
+      "secp256k1": true
```

Comment:
@rekmarks `secp256k1` is used by `ethereumjs-wallet` in one of the tests. Couldn't this be replaced by hardcoded test vectors instead? Is there a need to actually call the library in the tests?

Reply:
Yeah, this is just an accident of history where I first tested against other libraries and later added the actual test vectors. Do you mind creating an issue for this so we can track it and someone can fix it later?

Reply:
Will do a PR for this!